lib/src/chart/base/base_chart/base_chart_data.dart

```diff
@@ -111,6 +111,13 @@ class FlTitlesData {
   }
 }
 
+/// this is mimic of [MainAxisAlignment] to aligning the [SideTitles]
+enum SideTitlesAlignment {
```

Comment:
The default position of `TextPainter` is center of it's x-axis point. 
I just add this alignment for whom wanna set this view for better rendering in special cases.
If you think it's an irrelevant parameter I can remove it.

Reply:
Please let's remove it because it is not fully customizable, we can just customize the x-axis pivot, and for sure we should add customization for both axes.
We can add it in the future if it is needed.
Thanks!

Reply:
that's a good point, I just removed it.